subsystems/intakesubsystem.py

```diff
@@ -0,0 +1,74 @@
+from commands2 import SubsystemBase
+from wpilib import PneumaticsModuleType, Solenoid, SmartDashboard
+import constants
+from util.helpfulIO import Falcon
+
+
+class IntakeSubsystem(SubsystemBase):
+    def __init__(self) -> None:
+        SubsystemBase.__init__(self)
+        self.setName(__class__.__name__)
+        self.intakeDeployed = False  # default to intake retracted and not reversed
```

Comment:
Feels like intakesubsystem should get the same revamp that indexer did, with enums and stuff to make it more organized. I can do this, but anyone have rejections to that?

Reply:
Alright, I will give it a shot sometime later today based off of indexer's functionality. Shouldn't be to much of a problem as this is my expertise at this point (copying code and changing it slightly to work on other stuff)